src/modules/sih/aero.hpp

```diff
@@ -262,6 +271,12 @@ class AeroSeg
 	// must be called after update_aero()
 	matrix::Vector3f get_Ma() const { return _Ma; }
 
+	// return the velocity in segment frame
+	matrix::Vector3f get_vS() const { return _v_S; }
+
+	// copy assignment operator
+	AeroSeg& operator=(const AeroSeg&){return *this;}
```

Comment:
This does not actually assign.

Reply:
we can leave that, I'm finally not using it.